app/views/admin/invitations/index.html.haml

```diff
@@ -0,0 +1,79 @@
+- content_for :page_title do
+  Invitations
+
+- content_for :menu do
+  .item.hidden{ data: { behavior: "bulk-edit-menu" } }
+    .divider
+    .header Bulk Actions
+    .menu
+      .item
+        .ui.fluid.icon.button
+          %i.hide.icon
+          disable all
+
+      .item
+        .ui.fluid.icon.negative.basic.button
+          %i.trash.icon
+          delete all
+
+
+- content_for :body do
+  .ui.two.wide.column
+    .ui.row
+      = render partial: "menu", locals: { count: @count }
+
+  .ui.fourteen.wide.column
```

Comment:
what is this `.fourteen.wide.column` even I dont

Reply:
but seriously what is this css framework

Reply:
Semantic UI. Not 100% a fan of some of the css names but it works well for me https://semantic-ui.com/ and I like the prebuilt JS modules they have like the search and sidebar stuff.